briefings.html.haml

```diff
@@ -111,6 +111,15 @@ title: Events & Briefings
           %p Catch up on anything you might have missed.
         .container.wow.fadeInUp
           .col-sm-6.col-md-12
+              .service-02
```

Comment:
The spacing is not quite right - I think you need to take two whitespace characters out from the front of this because right now it is at the same level as the line below it.